pkg/api/customization/clusterregistrationtokens/formatter.go

```diff
@@ -33,9 +40,14 @@ func Formatter(request *types.APIContext, resource *types.RawResource) {
 			image.Resolve(settings.AgentImage.Get()),
 			getRootURL(request),
 			token,
-			ca)
+			caNonWindows)
 		resource.Values["token"] = token
 		resource.Values["manifestUrl"] = url
+		resource.Values["windowsNodeCommand"] = fmt.Sprintf(windowsNodeCommandFormat,
+			image.Resolve(settings.AgentImage.Get()),
```

Comment:
This should be Setting.WindowsAgentImage?  setting.AgentImage just return a `rancher/rancher-agent:version`

Reply:
nvm, i saw you put the prefix. But it would be better to add a setting for that.

Reply:
I tried that before, however `setting.AgentImage.Get()` could return the shortly commit hash matching rancher server, just like: `rancher/rancher-agent:1b22cb9`, but `setting.WindowsAgentImage.Get()` didn't. So I committed the current way.

Reply:
You have to define it. It is fine that this is hard-coded now, but as long as window agent image gets in its own release cycle we have to put a setting like that.